docs/Upgrading-2.0-to-3.0.md

```diff
@@ -353,7 +353,7 @@ As said, we've added a new optional RPC service for improved light client suppor
 @@ -49,6 +49,7 @@ use sp_consensus::SelectChain;
  use sp_consensus_babe::BabeApi;
  use sc_rpc::SubscriptionTaskExecutor;
- use sp_transaction_pool::TransactionPool;
```

Comment:
Isn't 3.0 already released? We probably shouldn't be updating that.

Reply:
have a Upgrading-3.0-to-master.md?